repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/mocks/MockIdentityService.groovy

```diff
@@ -587,7 +587,7 @@ class MockIdentityService {
             """{
   "RAX-KSGRP:groups": [
     {
-        "id": "0",
+        "id": "107",
```

Comment:
I probably should have left this as 0 and configured that to be used for the test. This may break other tests. Again, this was mostly just used to diagnose what was going on.